Medical-Units-MSSC/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Most likely we don't need that, let's check if all works without it and remove everything related to mvnw

Reply:
I will try remove it after finish appointment.